### PR TITLE
RELATED: ONE-4583 Drill intersection reports parent attributes

### PR DESCRIPTION
**Pull request by pbenes**
Pivot table now reports all preceding row attributes when performing drill
on certain row attribute. The last item in the intersection array is the
attribute header (+item) corresponding to the clicked cell.

Also should fix RAIL-2557

---

Supported PR commands:

| Command         | Description            |
| --------------- | ---------------------- |
| `ok to test`    | Re-run standard checks |
| `extended test` | BackstopJS tests       |

---

# PR Checklist

-   [ ] commit messages adhere to the [commit message guidelines](docs/contributing.md#what-should-the-commits-look-like)
-   [ ] `check` passes
-   [ ] `check-extended` passes
-   [ ] `rush change` [was run if applicable](docs/contributing.md#how-do-i-describe-my-changes-for-the-changelog)

### CODE REVIEW

**Comment by gdgate** [comment]
Merge Failed.

Zuul merger could not merge this change into the base branch. This is most likely caused by merge conflicts. Please rebase the change and upload the rebased version. In case of further errors, contact the zuul administrator.

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/232/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/1821/ : FAILURE in 11m 00s

File: libs/sdk-ui-tests/scenarios/pivotTable/drilling.tsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+export default scenariosFor<IPivotTableProps>("PivotTable", PivotTable)
+    .withGroupNames("drilling")
+    .withVisualTestConfig({ screenshotSize: { width: 1200, height: 800 } })
+    .withDefaultTags("vis-config-only", "mock-no-scenario-meta")
+    .addScenario("Pivot table with drill on all row atributes", {
+        ...PivotTableWithTwoMeasuresAndThreeRowsAndTwoCols,
+        drillableItems: [ProductPredicate, DepartmentPredicate, SalesRepPredicate],
+        onDrill: action("onDrill"),
+    })
+    .addScenario("Pivot table with drill on all row attributes and measures", {
```
Comment (lupko) [inline libs/sdk-ui-tests/scenarios/pivotTable/drilling.tsx:23]: I think you don't need the 'Pivot Table' prefix here. the infra will add automatically

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/235/ : SUCCESS in 3s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/1824/ : SUCCESS in 23m 27s

File: libs/sdk-ui-tests/scenarios/pivotTable/drilling.tsx (hunk 1 above)
Comment (pbenes) [reply libs/sdk-ui-tests/scenarios/pivotTable/drilling.tsx]: You're right, updated (It's correctly put to url and in the tree it is not in the story title but defined by the hierrarchy listing)

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/237/ : SUCCESS in 3s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/1826/ : SUCCESS in 11m 43s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/239/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/1828/ : SUCCESS in 14m 14s

**Comment by pbenes** [comment]
extended test

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/241/ : SUCCESS in 3s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-backstop-tests/219/ : UNSTABLE in 19m 48s

**Comment by pbenes** [comment]
extended test

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/242/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/1831/ : SUCCESS in 14m 01s

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/243/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-backstop-tests/220/ : SUCCESS in 18m 28s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/244/ : SUCCESS in 5s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/1832/ : SUCCESS in 18m 47s

**Comment by no23reason** [comment]
extended test

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/249/ : SUCCESS in 3s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-backstop-tests/221/ : SUCCESS in 11m 03s

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-fossa-scanning-zuul-el7/214/ : SUCCESS in 20s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-merge-compliance-check/469/ : SUCCESS in 11s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/1839/ : SUCCESS in 14m 20s

**Pull request merged**